index.html

```diff
@@ -26,24 +24,259 @@ <h1>Hey there. I'm Richard</h1>
          </div>
           <div class="support">
             <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
-  don’t hestiate to contact me.</p>
+          don’t hestiate to contact me.</p>
           </div>
           <div class="social-media">
             <a href="#"><i class="fab fa-twitter social"></i></a>
             <a href="#"><i class="fab fa-linkedin social"></i></a>
             <a href="#"><i class="fab fa-medium-m social"></i></a>
-            <!-- <a href="#"><i class="fab fa-medium social"></i></a> -->
             <a href="#"><i class="fab fa-github social"></i></a>
             <a href="#"><i class="fab fa-angellist social"></i></a>
-
           </div>
-
           <div class="arrow">
             <button  class="arrow-button" ><i class="fas fa-chevron-down fa-2x"></i></button>
           </div>
         </section>
+   </div>
+   <div class="grid-container">
```

Comment:
- [ ] The container of the section should a `section` tag. Please change it for good practice. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Use%20the%20appropriate%20tags%20for%20each%20element%20(e.g.%2C%20links%2C%20titles%2C%20etc.)%20and%20use%20HTML5%20semantic%20tags%20(e.g.%2C%20header%2C%20nav)%20over%20divs.).